### PR TITLE
Reseq core

### PR DESCRIPTION
**Pull request by mhsieh**

### CODE REVIEW

File: mk/travis.mk [hunk 1]
```diff
@@ -7,7 +7,7 @@ HAVE_PYTHON =
 HAVE_CMAKE  =
 HAVE_HDF5   = ${HOME}/hdf5-1.8.13-linux-x86_64-shared
 HAVE_BOOST  = ${HOME}/boost_1_56_0
-DEBUG       = yes
-VERBOSE     = 1
-export DEBUG
-export VERBOSE
+#DEBUG       = yes
+#VERBOSE     = 1
```
Comment (pb-cdunn) [inline mk/travis.mk]: Won't it be hard to debug problems in Travis if this is suppressed?

Comment (mjhsieh) [inline mk/travis.mk]: it stops when the console log exceed certain limit.

**Comment by pb-cdunn** [comment]
I don't know about DEBUG vs. VERBOSE. But seeing only the commands issued by make-rules shouldn't be all that much info.

Independent of this, it would be slick if activity/tee-to-log were driven by a separate script which could actually *display* the log on failure. We can't just login to Travis and look for the build failure.

**Comment by mhsieh** [comment]
with bin/activity out of the way, DEBUG no longer functional, we can get rid of it.

**Comment by pb-cdunn** [comment]
That's a nice change!

But here is a new problem:
```
make[1]: Entering directory `/tmp/cdunn/pitchfork/ports/thirdparty/gmap'
curl -L -O http://research-pub.gene.com/gmap/src/gmap-gsnap-2015-12-31.v7.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 4072k  100 4072k    0     0  1236k      0  0:00:03  0:00:03 --:--:-- 1236k
md5sum -c MD5SUM || exit
gmap-gsnap-2015-12-31.v7.tar.gz: OK
tar zxf gmap-gsnap-2015-12-31.v7.tar.gz -C /tmp/cdunn/pitchfork/workspace
(cd /tmp/cdunn/pitchfork/workspace/gmap-2015-12-31 && \
        ./configure --prefix=/tmp/cdunn/pitchfork/staging/gmap-2015-12-31) >& config.log || /tmp/cdunn/pitchfork/bin/diewith config.log
make -C /tmp/cdunn/pitchfork/workspace/gmap-2015-12-31 >& tee build.log || /tmp/cdunn/pitchfork/bin/diewith build.log
cat: build.log: No such file or directory
```

**Comment by mhsieh** [comment]
fixed.

**Pull request merged**